src/Platforms/AbstractPlatform.php

```diff
@@ -220,6 +222,18 @@ public function getVarcharTypeDeclarationSQL(array $column)
         return $this->getVarcharTypeDeclarationSQLSnippet($column['length'], $fixed);
     }
 
+    /**
+     * Returns the SQL snippet used to declare a string column type.
+     *
+     * @param mixed[] $column
+     *
+     * @return string
+     */
+    public function getStringTypeDeclarationSQL(array $column)
```

Comment:
I think it's OK to use a native return type declaration rightaway, isn't it?

```suggestion
     */
    public function getStringTypeDeclarationSQL(array $column): string
```

Reply:
It proxies the method that doesn't have a return type hint, so I'd leave it as is.

Reply:
The proxied method is supposed to return a string. If it does not, don't we want to learn about that as soon as possible? We will either have a bug to fix or  better knowledge about the actual return type that should be documented on the proxied method.

Reply:
This way of thinking could be applied to any method that has its return type documented but not enforced. In this specific case, the code returning a non-string may be implemented in a 3rd-party platform. I don't believe this change should be made as part of deprecation.

Reply:
Ok then